sdks/java/expansion-service/src/test/java/org/apache/beam/sdk/expansion/service/ExpansionServerTest.java

```diff
@@ -45,4 +47,20 @@ public void testHostPortAvailableAfterClose() throws Exception {
     assertThat(expansionServer.getHost(), is("localhost"));
     assertThat(expansionServer.getPort(), greaterThan(0));
   }
+
+  @Test
```

Comment:
Do you also need an end-to-end integration test to make sure that "PROCESS" mode works for cross-language transforms on Flink ? (can be a separate PR).

Reply:
Hm, I thought there was one. :)
If there is no such test, I think it would be good to have one. I'll create a Jira for that.

Reply:
I think current tests run with Docker mode.